tests/fixtures/rdf.py

```diff
@@ -1,30 +1,55 @@
 """Test fixtures to set up a test named graph for loading RDF data."""
 import os
+from flask import config
 
 import pytest
 import requests
 from requests.auth import HTTPDigestAuth
 
 
-def get_sparql_auth_conf(config_obj) -> dict:
-    """Fetch SPARQL auth configuration from the configuration object."""
+def get_sparql_auth_conf() -> dict:
```

Comment:
Can we investigate a better way of passing these in? These if statements may cause problems in the future. Some suggestions:

1. Change the return type from a dict to a python obj e.g. namedtuple or dataclass. This ensures stricter handling of the sparql auth items.
2. Instead of having 4 different params that we save in a configuration file, can we instead create one of the from `user@password:uri_def. We follow a similar pattern in our other configs e.g. https://github.com/jnduli/genenetwork3/blob/52cd294c2f1d06dddbd6ff613b11f8bc43066038/gn3/db_utils.py#L13
3. Since we're loading data into virtuoso from start, can we also launch a custom virtuoso instance here using `subprocess`?